### PR TITLE
node 9.8.0

### PR DESCRIPTION
**Pull request by DomT4**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/node.rb [hunk 1]
```diff
@@ -105,7 +104,7 @@ def post_install
       prefix = #{HOMEBREW_PREFIX}
       python = /usr/bin/python\n
     EOS
-    (node_modules/"npm"/"npmrc").atomic_write npmrc
```
Comment (ilovezfs) [inline Formula/node.rb]: Again with unrelated style changes …

Comment (DomT4) [reply Formula/node.rb]: Well since I can no longer enforce my style onto the core via direct commit I've got to try and satisfy the habit somehow.

**Review by ilovezfs — APPROVED** [review]

**Comment by commitay** [comment]
Thanks @DomT4!
